build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -994,6 +1009,78 @@ message GetTreeResponse {
   string next_page_token = 2;
 }
 
+// A request message for
+// [Capabilities.GetCapabilities][google.devtools.remoteexecution.v1test.Capabilities.GetCapabilities].
+message GetCapabilitiesRequest {
+  // The instance of the execution system to operate against. A server may
+  // support multiple instances of the execution system (with their own workers,
+  // storage, caches, etc.). The server MAY require use of this field to select
+  // between them in an implementation-defined fashion, otherwise it can be
+  // omitted.
+  string instance_name = 1;
+}
+
+// A response message for
+// [Capabilities.GetCapabilities][google.devtools.remoteexecution.v1test.Capabilities.GetCapabilities].
+message GetCapabilitiesResponse {
```

Comment:
In general, the heavy nesting of messages here makes it a bit difficult for me to follow. Is there a reason not to inline all these messages into the top-level one, except perhaps for PriorityCapabilities?

Reply:
Yes, the service provides remote caching and remote execution and I wanted to separate the properties of each into their own messages, as, for example, a cache-only server will simply not provide any ExecutionCapabilities and that will be clear (I thought). Also, they both have digest function, and it doesn't have to be the same one (there may be more than one for the cache, but just one for execution).
One thing I was not sure in the beginning is having UpdateActionCacheCapabilities as a message -- Eric suggested that (IIRC), because there will probably be more fields related to only that and it would make sense to encapsulate them, and if the message missing, that means no such capability entirely.